src/components/AboutMe.tsx

```diff
@@ -54,22 +145,30 @@ export default function AboutMe() {
                     <div className="mx-auto max-w-md px-4 sm:max-w-2xl sm:px-6 lg:px-0 lg:py-20 lg:max-w-none">
                         <blockquote>
                         <div>
-                            <svg
-                            className="h-12 w-12 text-light-brown"
-                            fill="currentColor"
-                            viewBox="0 0 32 32"
-                            aria-hidden="true"
-                            >
-                            <path d="M9.352 4C4.456 7.456 1 13.12 1 19.36c0 5.088 3.072 8.064 6.624 8.064 3.36 0 5.856-2.688 5.856-5.856 0-3.168-2.208-5.472-5.088-5.472-.576 0-1.344.096-1.536.192.48-3.264 3.552-7.104 6.624-9.024L9.352 4zm16.512 0c-4.8 3.456-8.256 9.12-8.256 15.36 0 5.088 3.072 8.064 6.624 8.064 3.264 0 5.856-2.688 5.856-5.856 0-3.168-2.304-5.472-5.184-5.472-.576 0-1.248.096-1.44.192.48-3.264 3.456-7.104 6.528-9.024L25.864 4z" />
-                            </svg>
                             <p className="mt-6 text-2xl font-medium text-white">
-                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed urna nulla vitae laoreet augue. Amet
-                            feugiat est integer dolor auctor adipiscing nunc urna, sit.
+                                I have experience in the following front-end tools:
+                                <div className="flex space-x-6 md:order-2 py-4 pb-8">
+                                    {frontendItems.map((item) => (
+                                        <a key={item.name} href={item.href} className="hover:text-light-brown transition-all duration-300">
```

Comment:
I don't think is necessary to put the link on the tech because the user of your site probably knows about it, but sounds okay. what do you think put the `target="_blank"`? I think is better because the site will not be replaced anymore.

Reply:
Wow, this is very cool. Thanks for the hint, I was really angry with the replacement.